src/Microsoft.Sbom.Api/Providers/ExternalDocumentReferenceProviders/CGExternalDocumentReferenceProvider.cs

```diff
@@ -74,7 +74,7 @@ protected override (ChannelReader<JsonDocWithSerializer> results, ChannelReader<
 
         protected override (ChannelReader<ScannedComponent> entities, ChannelReader<FileValidationResult> errors) GetSourceChannel()
         {
-            var (output, cdErrors) = sbomComponentsWalker.GetComponents(Configuration.BuildComponentPath?.Value);
+            var (output, cdErrors) = sbomComponentsWalker.GetComponents(Configuration.BuildComponentPath?.Value ?? Configuration.BuildDropPath?.Value);
```

Comment:
This will use the build drop path for the component scanning, we don't want that. We want to only scan containers but not other components

Reply:
So we would need a separate parameter to use as the desired output path? Maybe add a '-o' parameter similar to how we do it in the Validation args?